src/test/java/org/opentripplanner/middleware/triptracker/ManageLegTraversalTest.java

```diff
@@ -670,7 +774,40 @@ void testGetDistanceToEndOfLeg() {
 
         assertTrue(TravelerLocator.getDistanceToEndOfLeg(travelerPosition, GMAP_UPCOMING_RADIUS) <= GMAP_UPCOMING_RADIUS);
     }
-    
+
+    @Test
+    void testGetDistanceToEndOfLeg2() {
```

Comment:
This is very similar to the test above. Could a parameterized test be created to cover both?